Train.py

```diff
@@ -76,14 +76,12 @@
                 print('ctr=' + str(data_index.ctr))
                 print('epoch progress=' + str(100 * data_index.ctr /
                                               len(data_index.all_steer)) + '%')
-
                 if args.display:
                     batch.display()
-
                     plt.figure('loss')
                     plt.clf()  # clears figure
                     loss_record['train'].plot('b')  # plot with blue color
                     loss_record['val'].plot('r')  # plot with red color
                     print_timer.reset()
 
-            batch = Batch.Batch(net)  # Reinitialiize batch
+            batch = Batch.Batch(net)  # Reinitialiize batch. ***Why is this necessary?***
```

Comment:
This line serves to replace your batch.clear() in the previous version (pytorch2). I'll look into what that did, but I changed it from .clear to a re-initialization since it essentially does the same thing.